test/e2e/app-dir/app-basepath/index.test.ts

```diff
@@ -69,7 +69,7 @@ describe('app dir - basepath', () => {
     })
   })
 
-  it.each(['/base/refresh', '/base/refresh?foo=bar'])(
+  it.each(['/base/refresh?foo=bar'])(
```

Comment:
Since it's only one case now it can be changed to just `it`

Reply:
whoops, that was just for debugging....
```suggestion
  it.each(['/base/refresh', '/base/refresh?foo=bar'])(
```